projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/graphql/it/GraphQLExecutionsIT.java

```diff
@@ -0,0 +1,203 @@
+   }
+
+   @Test
+   public void testExecutions_filterByEndTimeGte() throws Exception {
+      mockMvc.perform(MockMvcRequestBuilders.get(JOBS_URI)
+                  .queryParam("query", getQuery())
+                  .param("variables", "{" +
+                        "\"filter\": {" +
+                        "      \"endTimeGte\": \"" + dataJobExecution2.getEndTime() + "\"" +
+                        "    }," +
+                        "\"pageNumber\": 1," +
+                        "\"pageSize\": 10" +
+                        "}")
+                  .with(user("user")))
+            .andExpect(status().is(200))
+            .andExpect(content().contentType("application/json"))
+            .andExpect(jsonPath(
+                  "$.data.content[*].id",
+                  Matchers.contains(dataJobExecution1.getId(), dataJobExecution2.getId())))
+            .andExpect(jsonPath(
+                  "$.data.content[*].jobName",
+                  Matchers.contains(dataJob1.getName(), dataJob2.getName())))
+            .andExpect(jsonPath(
+                  "$.data.content[*].status",
+                  Matchers.contains(dataJobExecution1.getStatus().toString(), dataJobExecution2.getStatus().toString())))
+            .andExpect(jsonPath(
+                  "$.data.content[*].id",
+                  Matchers.not(Matchers.contains(dataJobExecution3.getId()))));
+   }
+
+   @Test
+   public void testExecutions_filterByStatusIn() throws Exception {
+      mockMvc.perform(MockMvcRequestBuilders.get(JOBS_URI)
+                  .queryParam("query", getQuery())
+                  .param("variables", "{" +
+                        "\"filter\": {" +
+                        "      \"statusIn\": [\"" + dataJobExecution1.getStatus().toString() + "\", \"" + dataJobExecution2.getStatus().toString() + "\"]" +
+                        "    }," +
+                        "\"pageNumber\": 1," +
+                        "\"pageSize\": 10" +
+                        "}")
+                  .with(user("user")))
+            .andExpect(status().is(200))
+            .andExpect(content().contentType("application/json"))
+            .andExpect(jsonPath(
+                  "$.data.content[*].id",
+                  Matchers.contains(dataJobExecution1.getId(), dataJobExecution2.getId())))
+            .andExpect(jsonPath(
+                  "$.data.content[*].jobName",
+                  Matchers.contains(dataJob1.getName(), dataJob2.getName())))
+            .andExpect(jsonPath(
+                  "$.data.content[*].status",
+                  Matchers.contains(dataJobExecution1.getStatus().toString(), dataJobExecution2.getStatus().toString())))
+            .andExpect(jsonPath("$.data.content[*].id", Matchers.not(Matchers.contains(dataJobExecution3.getId()))));
+   }
+
+   private DataJobExecution createDataJobExecution(
+         String executionId,
+         DataJob dataJob,
+         OffsetDateTime startTime,
+         OffsetDateTime endTime,
```

Comment:
I'm wondering if we should refactor and put this method in a util package since a lot of graphql these IT tests use (and will use) a variation of this method.

Reply:
Done